api/resolvers/notifications.js

```diff
@@ -234,10 +234,14 @@ export default {
             FROM "Invoice"
             WHERE "Invoice"."userId" = $1
             AND "confirmedAt" IS NOT NULL
-            AND "isHeld" IS NULL
             AND "actionState" = 'PAID'
             AND "actionType" = 'TRANSFER'
             AND created_at < $2
+            AND NOT EXISTS (
+              SELECT 1
+              FROM "InvoiceForward"
+              WHERE "InvoiceForward"."invoiceId" = "Invoice".id
+            )              
```

Comment:
if there is an InvoiceForward, it must be a p2p transfer, this is to avoid showing the notification twice (for deposit + p2p forward)